yarn.lock

```diff
@@ -1359,9 +1359,9 @@
     identity-obj-proxy "^3.0.0"
 
 "@jahia/ui-extender@^1.0.0":
-  version "1.0.0"
-  resolved "https://npm.jahia.com/@jahia%2fui-extender/-/ui-extender-1.0.0.tgz#2a5ff8d63f200c00ffe437c39241d7bf5e6493df"
-  integrity sha512-9d6sM/j/CaBcxUPIJm/IAyOehJ/ZxOpIhvYooysM75czWHPZVCbaMCE+VHzdP2UztINw3ezr0DYKcxm7oDnfNQ==
+  version "1.0.3"
```

Comment:
this is upgraded on yarn.lock, but not in dependencies?

Reply:
Yes when I ran yarn it automatically bump the version inside the yarn.lock